pkg/sql/executor.go

```diff
@@ -899,11 +923,11 @@ func (e *Executor) execParsed(
 				}
 			}
 
+			// TODO(tohlson): allow schema change errors
 			// Exec the schema changers (if the txn rolled back, the schema changers
 			// will short-circuit because the corresponding descriptor mutation is not
 			// found).
-			session.tables.releaseTables(session.Ctx())
-			txnState.schemaChangers.execSchemaChanges(session.Ctx(), e, session, res.ResultList)
+			txnState.schemaChangers.execSchemaChanges(session.Ctx(), e, session, ResultList{})
```

Comment:
This is probably where I need help the most. Schema changes seem quite gross to deal with since in `execSchemaChanges` they modify past results. Seems like I should be able to just send an error saying that the schema change actually failed, even if I've previously said it succeeded.